README.md

```diff
@@ -1,11 +1,158 @@
-## My Project
+# OpenSearch OCI Object Storage Plugin
 
-TODO: Fill this README out!
+## Build and install the plugin
+To build the plugin zip distribution file
+```bash
+mvn clean install -PAdvanced
+```
+Install the plugin on your Elasticsearch cluster
+```bash
+OS_HOME=<YOUR OPENSEARCH INSTALLATION PATH HERE> # (e.g. /Users/saherman/opensearch)
```

Comment:
Recommend expanding `OS` into `OPENSEARCH` because OS is too operating system-y.